CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(pennylane_lightning)
+
+# Read and set pennylane_lightning version
+execute_process(COMMAND "/usr/bin/env" "python"  "-c"
+	"with open('${CMAKE_SOURCE_DIR}/pennylane_lightning/_version.py') as f: print(f.readlines()[-1].split()[-1].strip(\"\\\"'\"), end='')"
```

Comment:
Thank you for your feedback! Of course, the File command is better for consistency, but I called python because handling a file using CMake becomes easily tedious. Still, let me try a bit more to find a compact CMake only way to do this.

Reply:
I added a pure CMake function that extract version from `_version.py` file in 14b2cfd8d48597e7d6b0644d5936cbecea01b349. Does it seem nice? I don't know if there is any better way to do this.

Reply:
Yep, I'm good with this.